lint.sh

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I do use lint.sh fairly.  Tests and npm run build take so long that I doubt I'll just want to do run_tests.sh instead.  Any way to keep this separate still?

Reply:
i'll just not delete it :)